### DISCUSSION TITLE
Where is the api.md file?

### DISCUSSION THREAD

**Discussion by vicariousdrama** [ACINQ/phoenixd#54] [Q&A]
Unclear which repository the api.md for phoenix server resides that drives https://phoenix.acinq.co/server/api

The operation `POST /createinvoice` would benefit by documenting the descriptionHash option and denoting it as either description or descriptionHash should be provided.

ref: dec1c59c807a1b5469c8f42a99f7219caa9b352b

**Comment by dpad85** [comment]
The website has been updated, thank you for reporting this issue.
